README.rst

```diff
@@ -0,0 +1,184 @@
+|Build Status| |Documentation Status| |codecov|
```

Comment:
Pypi can apparently render .rst files as the extended documentation on the download page of the package, which I thought would be easier.